src/components/cards/cards.jsx

```diff
@@ -0,0 +1,284 @@
+// so that the mouseup is not swallowed by the iframe.
+const VideoStep = ({video, dragging}) => (
+    <div className={styles.stepVideo}>
+        {dragging ? (
+            <div className={styles.videoCover} />
+        ) : null}
+        <iframe
+            allowFullScreen
+            allow="autoplay; encrypted-media"
+            frameBorder="0"
+            height="337"
+            src={`${video}?rel=0&amp;showinfo=0`}
+            width="600"
+        />
+    </div>
+);
+
+VideoStep.propTypes = {
+    dragging: PropTypes.bool.isRequired,
+    video: PropTypes.string.isRequired
+};
+
+const ImageStep = ({title, image}) => (
+    <Fragment>
+        <div className={styles.stepTitle}>
+            {title}
+        </div>
+        <div className={styles.stepImageContainer}>
+            <img
+                className={styles.stepImage}
+                draggable={false}
+                src={image}
+            />
+        </div>
+    </Fragment>
+);
+
+ImageStep.propTypes = {
+    image: PropTypes.string.isRequired,
+    title: PropTypes.node.isRequired
+};
+
+const NextPrevButtons = ({onNextStep, onPrevStep}) => (
+    <Fragment>
+        {onNextStep ? (
+            <div>
+                <div className={styles.rightCard} />
+                <div
+                    className={styles.rightButton}
+                    onClick={onNextStep}
+                >
+                    <img
+                        draggable={false}
+                        src={nextIcon}
+                    />
+                </div>
+            </div>
+        ) : null}
+        {onPrevStep ? (
+            <div>
+                <div className={styles.leftCard} />
+                <div
+                    className={styles.leftButton}
+                    onClick={onPrevStep}
+                >
+                    <img
+                        draggable={false}
+                        src={prevIcon}
+                    />
+                </div>
+            </div>
+        ) : null}
+    </Fragment>
+);
+
+NextPrevButtons.propTypes = {
+    onNextStep: PropTypes.func,
+    onPrevStep: PropTypes.func
+};
+CardHeader.propTypes = {
+    onCloseCards: PropTypes.func.isRequired,
+    onShowAll: PropTypes.func.isRequired,
+    step: PropTypes.number,
+    totalSteps: PropTypes.number
+};
+
+const PreviewsStep = ({deckIds, content, onActivateDeckFactory, onShowAll}) => (
+    <Fragment>
+        <div className={styles.stepTitle}>
+            <FormattedMessage
+                defaultMessage="More things to try!"
+                description="Title card with more things to try"
+                id="gui.cards.more-things-to-try"
+            />
+        </div>
+        <div className={styles.decks}>
+            {deckIds.map(id => (
+                <div
+                    className={styles.deck}
+                    key={`deck-preview-${id}`}
+                    onClick={onActivateDeckFactory(id)}
```

Comment:
Curious about this pattern — since you're defining a function either way, is there an advantage to passing in a factory like this vs doing it inline, like `onClick={() => onActivateDeck(id)}`?  I think both are fine, just curious about this choice.